readme.md

```diff
@@ -7,10 +7,12 @@
 - Ansible 2.x (http://docs.ansible.com/ansible/intro_installation.html)
 - Vagrant (https://www.vagrantup.com/docs/installation/)
 - NFS
+- Yarn
 
 ## Getting started
 1. Create a mapping in your hosts file from phpsw.dev to 192.168.42.10.
-2. ```vagrant up```
+1. Run `yarn install` and either `yarn run encore dev` or `yarn run encore dev --watch` to generate the front-end assets.
```

Comment:
Given Ansible is being used to provision the VM, `yarn install` should be done with ansible. Have a chat with @TomAdam on getting this sorted. 

Also I'm not sure how this fits into the generate website process. Is it something that gets done once, or every time we generate a web page or only when CSS changes?

Reply:
Every time we need to re-generate the assets, so I'd usually run this on as part of the deployment process. You'll still need to run the steps locally or on the VM though locally to generate them if you want to see CSS. 😄

Reply:
I've added another issue #24 about deployments, which would include this step if we want to use that.

Reply:
So is yarn installed as part of the ansible provision? If not it needs to be added (this is so we can test locally).

Reply:
I haven't added it, but I can if it's not there already.

Reply:
Added in 29fdb0a43e3c15e9103d236c4eba619afa20deed.